sandbox/snippets/101functions/storage/azure-table-storage.md

```diff
@@ -1,17 +1,66 @@
+---
+title: Azure Output Table Storage Bindings
+description: Output received content to a table storage.
+author: MaximRouiller
+manager: scottca
+keywords: 
+ms.topic: article
+ms.date: 08/3024/2017
+ms.author: marouill
+ms.devlang: csharp
+#ms.service: cognitive-services
+---
+
 ## Azure Table Storage
 
-Some description here
+Defining a binding between an Azure Function and a table in a storage account can be defined in two different ways. 
 
-```
-Some code
+The sample functions presented here do not include a trigger and do not work as-is.
+
+### Code declaration
```

Comment:
Per meeting convos, I believe we decided to avoid adding static headers and are going with only the title, takeaways, and read more headers. cc/ @anthonychu to confirm

Reply:
Yes. Please see an example here thanks:
https://github.com/MicrosoftDocs/sandbox/blob/101functions/sandbox/snippets/101functions/triggers/github-comment-webhook.md

Reply:
Are we only avoiding H3 or can I use H4?

Reply:
Mentioned this during the meeting, but also wanted to write this down:
We're avoiding all static headers, with the exception of the title.